.github/workflows/build-pull-request.yml

```diff
@@ -24,30 +24,30 @@ jobs:
     # release branch PRs are excluded
     # because any time a new library version is released to maven central it can fail
     # which requires unnecessary release branch maintenance, especially for patches
-    if: ${{ !startsWith(github.base_ref, 'release/') }}
```

Comment:
see https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#overview

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression.

and the quotes are needed because `!` means something special in yaml that I don't understand

Reply:
Nice, thanks!